pull.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
if it's at the root of the folder (which I think is a good idea), it must be a little more polished.

1. set -e
2. add an helper, with a `-h` option
3. must work without any argument (in that case, I would say print a confirmation message, then pull all images)
4. a better CLI : `./pull.sh -S NAME1,NAME2,NAME3` 

And a lazy, but totally acceptable at a first step, is to put it in `utils/scripts/`

Side question, why you didn't used it in the action ?

Reply:
To be honest I just made this script by quickly extracting the logic from the action. At this point I'd rather just remove it from this PR and reintroduce it as its own PR instead.